example.py

```diff
@@ -44,16 +45,24 @@ def __init__(self):
         
         # Get icons by character
         coffee_icon = qta.icon_by_char('fa', qta.charmap('fa.coffee'))
-        coffee_button = QtGui.QPushButton(coffee_icon, 'Black please!')
+        coffee_button = QtGui.QPushButton(coffee_icon, 'Access by character')
         
         # Render a label with this font
-        label = QtGui.QLabel(QtCore.QChar(0xf0f4))
+        label = QtGui.QLabel(QtCore.QChar(0xf19c) + ' ' + 'Label')
```

Comment:
@SylvainCorlay, I know you changed this to `unichr`, but is there any real use case (for the main users) for directly using the unicode number of the icon, as opposed to the just the name?